canvasapi/course.py

```diff
@@ -438,6 +439,41 @@ def create_late_policy(self, **kwargs):
 
         return LatePolicy(self._requester, late_policy_json["late_policy"])
 
+    def create_lti_resource_link(self, url, title=None, custom=None, **kwargs):
+        """
+        Create a new LTI resource link.
+
+        :calls: `POST /api/v1/courses/:course_id/lti_resource_links \
+        <https://canvas.instructure.com/doc/api/lti_resource_links.html#method.lti/resource_links.create>`_
+
+        :param course_id: The ID of the course.
+        :type course_id: `int`
```

Comment:
Remove this reference to `course_id`, since it's not part of the function's arguments (we get this automatically from `self.id`)